File: src/invoicing/InvoiceImpl.java
package invoicing;

import items.Item;

public class InvoiceImpl implements Invoiced {

    @Override
    public void printInvoice(Item item) {
        System.out.println("\n-------------------------Invoice-------------------------\n" +
                "Item: " + item.name + "\nPrice: " + item.price +
                "\n---------------------------------------------------------\n");
    }
}

File: src/invoicing/InvoiceImpl.kt
package invoicing

import items.Item

class InvoiceImpl : Invoiced {
    override fun printInvoice(item: Item) {
        println("""
    
    -------------------------Invoice-------------------------
    Item: ${item.name}
    Price: ${item.price}
    ---------------------------------------------------------
    
    """.trimIndent())
    }
}